src/test/java/junit/edu/umich/its/cpmtest/MigrationTaskServiceTest.java

```diff
@@ -128,4 +128,14 @@ public void testGoogleGroupsNamefromSiteIDWhenNoProperEmailIdIsInFeed(){
 		assertEquals("9bfee837-9b2c-4b6e-9e84-d77e9fbe10e9",googleGroupName);
 
 	}
+
+
+	@Test
+	public void testCheckForDotsInFilename(){
```

Comment:
Is there a test for a normal file name that just doesn't have an extension?  e.g "Happy" vs "Happy.doc"